src/lib/tabwidget/tabicon.cpp

```diff
@@ -113,4 +128,26 @@ void TabIcon::paintEvent(QPaintEvent* event)
         p.drawPixmap(r, m_animationPixmap, QRect(m_currentFrame * pixmapSize, 0, pixmapSize, pixmapSize));
     else
         p.drawPixmap(r, m_sitePixmap);
+
+#if QT_VERSION >= QT_VERSION_CHECK(5,7,0)
```

Comment:
This probably doesn't need ifdef? m_audioIconDisplayed should not ever be true on Qt < 5.7.0?